sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.azure.spring</groupId>
   <artifactId>spring-cloud-azure-dependencies</artifactId>
-  <version>4.5.0-beta.2</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-dependencies;current} -->
```

Comment:
the azure-sdk bom version in this file is not correct.

Reply:
Could you help check whether we have an action for upgrading azure-sdk-bom in our release todo list?

Reply:
Not find it, will add one.